test/e2e/mock/pvc.go

```diff
@@ -27,7 +28,7 @@ func (c *Client) CreatePVC(ctx context.Context, p string) (*v1.PersistentVolumeC
 					v1.ResourceStorage: resource.MustParse("10Gi"),
 				},
 			},
-			StorageClassName: &storageClassName,
+			StorageClassName: getMaxIOPSStorageClass(),
```

Comment:
No HDD availability anymore?

Reply:
I think that e2e test are still work-in-progress and require refactoring. I haven't go through those yet. We do deploy HDD storage classes with Kubernetes.